tests/Model/CachingIteratorTest.php

```diff
@@ -53,7 +53,7 @@ public function testIterationWithEmptySet()
 
     public function testPartialIterationDoesNotExhaust()
     {
-        $traversable = $this->getTraversableThatThrows([1, 2, new Exception()]);
+        $traversable = $this->getTraversable([1, 2, new Exception()]);
```

Comment:
I assume this is an easy change because we never iterate to the third element?

Reply:
Almost. The difference between the methods is that `getTraversableThatThrows` checks an element and throws it if it's an Exception. Since we don't need to iterate over exceptions anywhere, I decided to simplify this.